portfolio/src/main/webapp/script.js

```diff
@@ -22,8 +22,23 @@ function addMessage() {
   const responseContainer = document.getElementById('responseContainer');
   responseContainer.innerText = messages;
 }
-function getMessage() {
-  fetch('/response').then(response => response.text()).then((message) => {
-    document.getElementById('responseContainer').innerText = message;
-  });
-}
\ No newline at end of file
+
+function getServerString (){
+    fetch('/data').then(response => response.json()).then((res) => {
+    
+    const responseElement = document.getElementById('responseContainer');
+    responseElement.innerHTML = '';
+    responseElement.appendChild(
+        createListElement('Info: ' + res[0]));
+    responseElement.appendChild(
+        createListElement('Action: ' + res[1]));
+    responseElement.appendChild(
+        createListElement('Work: ' + res[0]));
+});
+}
+
+function createListElement(text) {
```

Comment:
This is such a good pattern to have. (No action, just wanted to call out I like this).